src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -47,14 +48,23 @@
           (assoc res :body {:token token}))
         res))))
 
+(defn- valid-url? [url]
+  (try
+    (URL. url)
+    true
+    (catch MalformedURLException _
+      false)))
```

Comment:
Voeg hier ajb een check toe dat het een http of https URL is. java.net.URL accepteerd ook een aantal andere protocollen.

```(java.net.URL. "file://foo") => #<java.net.URL@2457d389 file://foo>```

Reply:
ok